src/openforms/js/components/formio_builder/WebformBuilder.js

```diff
@@ -218,33 +337,27 @@ class WebformBuilder extends WebformBuilderFormio {
   // Taken from https://github.com/formio/formio.js/blob/v4.13.13/src/WebformBuilder.js#L1450
   // Modified so that copied components have the right defaultValue
   copyComponent(component) {
-    if (!window.sessionStorage) {
-      return console.warn('Session storage is not supported in this browser.');
-    }
-    this.addClass(this.refs.form, 'builder-paste-mode');
-    window.sessionStorage.setItem(
-      'formio.clipboard',
-      JSON.stringify(this.fixDefaultValues(component))
-    );
+    const fixedComponent = {...component, schema: fixDefaultValues(component)};
+    return super.copyComponent(fixedComponent);
```

Comment:
I'm not sure anymore, but I think I had avoided calling super because it does:
`window.sessionStorage.setItem('formio.clipboard', JSON.stringify(component.schema));`
And the `component.schema` then overwrites the fixes that we make here?

Reply:
I did break things in earlier versions, but while fixing that I think your concern is accounted for. The original version put the `fixedSchema` on the clipboard, this is still happening now because I mutate the component that's passed to the super call:

```js
const fixedComponent = {...component, schema: fixDefaultValues(component)};
```
so what gets put on the clipboard via `super.copyComponent(fixedComponent)` is essentially `JSON.stringify(fixDefaultValues(component))`

Reply:
Ahh yes, makes sense